Darius-Ndubi/Fast-Food-Fast
# Ft user history 160088014

####  What does this PR do?
  Shows the user profile endpoint

#### Description of Task to be completed?
- Allow the user to see all orders
- Allow the user to see pending orders
- Allow the user to see past orders
- Links  the user to add more orders

#### How should this be manually tested?
- Fork the [repo](https://github.com/Darius-Ndubi/Fast-Food-Fast)
- git clone https://github.com/Darius-Ndubi/Fast-Food-Fast.git
- cd Fast-Food-Fast
- git checkout ft-admin-see-orders-160087939
- open admin_profile.html in a browser
- https://darius-ndubi.github.io/user_profile.html

#### What are the relevant pivotal tracker stories?
[#160087586](https://www.pivotaltracker.com/story/show/160087586)
[#160088014](https://www.pivotaltracker.com/story/show/160088014)
